### ISSUE TITLE
Table of Contents -> Alphabetic Ordering

### ISSUE THREAD

**Issue by vontell** [hemanth/functional-programming-jargon#59]
It would be easier to use this repo if the table of contents / terms at the beginning of the main README.md file were listed in alphabetical order, in case the user is trying to search for a particular term.

**Comment by MichalZalecki** [comment]
It could be an *Index* at the end of the document referenced in ToC. I think it makes sense to keep ToC as it's right now so it reflects real order of covered topics.

**Comment by scycer** [comment]
I agree with keeping it in it's current order. I found it more important to explain them in the order they expand in complexity rather than alphabetical.
